docs/release/release.md

```diff
@@ -0,0 +1,430 @@
+# K3S Release Process
+## Setup Environment
+1. install Git (using any method that makes sense
+1. Configure Git for working with GitHub (add your ssh key, etc)
+1. install Go from binary
+1. set up default Go file structure
+   1. create $HOME/go/src/github.com/<your user>
+   1. create $HOME/go/src/github.com/k3s-io
+   1. create $HOME/go/src/github.com/rancher
+   1. create $HOME/go/src/github.com/rancherlabs
+   1. create $HOME/go/src/github.com/kubernetes
```

Comment:
I think creating these dirs before cloning will result in an error with the cloning

Reply:
Theses are the group directories, you would clone the corresponding projects into them. For instance, with k3s you would clone the k3s repo into the k3s-io directory.